From c179e6276c95ed63b5342afe0b2b9b84dd68a3fe Mon Sep 17 00:00:00 2001
From: Yannick Fertre <yannick.fertre@foss.st.com>
Date: Fri, 25 Mar 2022 13:55:27 +0100
Subject: [PATCH 393/529] ARM: configs: multi_v7_addons: add new panel for
 stm32mp1

Add support of new panel ROCKTECH HX8394 to multi_v7 addons
fragment file.

Signed-off-by: Yannick Fertre <yannick.fertre@foss.st.com>
Change-Id: Iad2ed976325e0f66010c0de3d6e7e31b0193f195
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/245665
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/284585
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -14,6 +14,7 @@ CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAUL
 CONFIG_DEBUG_PREEMPT=y
 CONFIG_DEBUG_SECTION_MISMATCH=y
 CONFIG_DNS_RESOLVER=y
+CONFIG_DRM_PANEL_ROCKTECH_HX8394=y
 CONFIG_DRM_SIMPLEDRM=y
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_FORCE_MAX_ZONEORDER=12
